recipes/stella-cv-fbow/all/conanfile.py

```diff
@@ -0,0 +1,120 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rm, rmdir
+
+required_conan_version = ">=1.53.0"
+
+
+class StellaCvFbowConan(ConanFile):
+    name = "stella-cv-fbow"
+    description = "FBoW (Fast Bag of Words) is an extremely optimized version of the DBoW2/DBoW3 libraries."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/stella-cv/FBoW"
+    topics = ("bag-of-words", "computer-vision", "visual-slam", "dbow")
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "fast_math": [True, False],
+        "avx": [True, False],
+        "mmx": [True, False],
+        "sse": [True, False],
+        "sse2": [True, False],
+        "sse3": [True, False],
+        "sse4": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "fast_math": True,
+        "avx": True,
+        "mmx": True,
+        "sse": True,
+        "sse2": True,
+        "sse3": True,
+        "sse4": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        if self.settings.arch not in ["x86", "x86_64"] or self.settings.compiler not in ["gcc", "clang", "apple-clang"]:
+            del self.options.avx
+            del self.options.mmx
+            del self.options.sse
+            del self.options.sse2
+            del self.options.sse3
+            del self.options.sse4
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        # https://github.com/stella-cv/FBoW/blob/master/include/fbow/vocabulary.h#L35
+        self.requires("opencv/4.9.0", transitive_headers=True, transitive_libs=True)
+        self.requires("llvm-openmp/17.0.6")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, 11)
```

Comment:
Shouldn't we also remove https://github.com/stella-cv/FBoW/blob/c6e3c29e3332a0b0834021797e2aa4e8eb66a3c1/CMakeLists.txt#L112?

Reply:
Done.